stories/0-Welcome.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
these stories came with storybook we can remove but they are useful boilerplate until we write our own.

Reply:
sounds good